api/control/v1alpha1/types.go

```diff
@@ -49,8 +49,8 @@ type BindingPolicySpec struct {
 	// WantSingletonReportedState indicates that (a) the number of selected locations is intended
 	// to be 1 and (b) the reported state of each downsynced object should be returned back to
 	// the object in this space.
-	// When multiple BindingPolicy objects match the same workload object,
-	// the OR of these booleans rules.
+	// For this field to take effect, it is required that no other binding policy with this field
+	// set to true selects a shared object.
```

Comment:
"a" here is not very clear. I think perhaps you mean something like the following.

> .. no other BindingPolicy with `WantSingletonReportedState == true` selects any of the same workload objects.

Reply:
Done. What you stated is right, the intention was to specify the "wanted" behavior, which would've been implemented later.